sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/RedundantJumpStatement.cs

```diff
@@ -115,5 +115,25 @@ void LongChain()
             else
             { }
         }
+
+        // https://github.com/SonarSource/sonar-csharp/issues/1265
+        void FalsePositive()
```

Comment:
_FalsePositive_ is a bit misleading - _RegressionTestForIssue1265_ might be better, and it means this class won't be found if we are doing a text search for false positives in the tests.